entity-service-client/src/main/java/org/hypertrace/entity/query/service/client/EntityLabelsClient.java

```diff
@@ -0,0 +1,27 @@
+package org.hypertrace.entity.query.service.client;
+
+import java.util.List;
+import java.util.Map;
+
+public interface EntityLabelsClient {
+  List<String> getEntityLabelsForEntity(String idColumnName,
+                                        String labelsColumnName,
+                                        String id,
+                                        String type,
+                                        Map<String, String> headers,
+                                        String tenantId);
+
+  Map<String, List<String>> getEntityLabelsForEntities(String idColumnName,
+                                                       String labelsColumnName,
+                                                       List<String> ids,
+                                                       String type,
+                                                       Map<String, String> headers,
+                                                       String tenantId);
+
+  List<String> getEntitiesWithLabels(String idColumnName,
```

Comment:
This sounds like it should be returning entities - I'm guessing these are IDs?

Reply:
Yes. These are IDs. Will rename